yw_website/website/templates/registration/login.html

```diff
@@ -1,47 +1,10 @@
+<html>
 {% load staticfiles %}
```

Comment:
might as well remove this line too since we aren't referencing the `staticfiles` directory anymore